com.hazelcast.jet.elastic.ElasticClientsTest.given_clientAsString_whenReadFromElasticSource_thenFinishSuccessfully

_master_ (commit b86829895be06f8152f61d96ca9d96439dd55d24)

Failed on Sonar bightly build: http://jenkins.hazelcast.com/job/jet-oss-master-sonar-nightly/604/testReport/com.hazelcast.jet.elastic/ElasticClientsTest/given_clientAsString_whenReadFromElasticSource_thenFinishSuccessfully/

Stacktrace:
```
java.lang.AssertionError: 

Expected size:<1> but was:<16> in:
<["{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.297Z","type":"indices_stats","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"indices_stats":{"_all":{"primaries":{"docs":{"count":1},"store":{"size_in_bytes":324},"indexing":{"index_total":1,"index_time_in_millis":3,"is_throttled":false,"throttle_time_in_millis":0},"search":{"query_total":0,"query_time_in_millis":0}},"total":{"docs":{"count":1},"store":{"size_in_bytes":324},"indexing":{"index_total":1,"index_time_in_millis":3,"is_throttled":false,"throttle_time_in_millis":0},"search":{"query_total":0,"query_time_in_millis":0}}}}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.297Z","type":"index_stats","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"index_stats":{"index":"my-index","total":{"docs":{"count":1},"store":{"size_in_bytes":324,"throttle_time_in_millis":0},"indexing":{"index_total":1,"index_time_in_millis":3,"throttle_time_in_millis":0},"search":{"query_total":0,"query_time_in_millis":0},"merges":{"total_size_in_bytes":0},"refresh":{"total_time_in_millis":25},"query_cache":{"memory_size_in_bytes":0,"hit_count":0,"miss_count":0,"evictions":0},"fielddata":{"memory_size_in_bytes":0,"evictions":0},"segments":{"count":1,"memory_in_bytes":2042,"terms_memory_in_bytes":1510,"stored_fields_memory_in_bytes":312,"term_vectors_memory_in_bytes":0,"norms_memory_in_bytes":128,"points_memory_in_bytes":0,"doc_values_memory_in_bytes":92,"index_writer_memory_in_bytes":0,"version_map_memory_in_bytes":0,"fixed_bit_set_memory_in_bytes":0},"request_cache":{"memory_size_in_bytes":0,"evictions":0,"hit_count":0,"miss_count":0}},"primaries":{"docs":{"count":1},"store":{"size_in_bytes":324,"throttle_time_in_millis":0},"indexing":{"index_total":1,"index_time_in_millis":3,"throttle_time_in_millis":0},"search":{"query_total":0,"query_time_in_millis":0},"merges":{"total_size_in_bytes":0},"refresh":{"total_time_in_millis":25},"query_cache":{"memory_size_in_bytes":0,"hit_count":0,"miss_count":0,"evictions":0},"fielddata":{"memory_size_in_bytes":0,"evictions":0},"segments":{"count":1,"memory_in_bytes":2042,"terms_memory_in_bytes":1510,"stored_fields_memory_in_bytes":312,"term_vectors_memory_in_bytes":0,"norms_memory_in_bytes":128,"points_memory_in_bytes":0,"doc_values_memory_in_bytes":92,"index_writer_memory_in_bytes":0,"version_map_memory_in_bytes":0,"fixed_bit_set_memory_in_bytes":0},"request_cache":{"memory_size_in_bytes":0,"evictions":0,"hit_count":0,"miss_count":0}}}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.305Z","type":"cluster_stats","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"cluster_name":"elasticsearch","version":"5.6.16","license":{"status":"active","uid":"c5e6c539-f52c-430d-a2fc-516ef594fb64","type":"trial","issue_date":"2020-09-02T13:47:52.571Z","issue_date_in_millis":1599054472571,"expiry_date":"2020-10-02T13:47:52.571Z","expiry_date_in_millis":1601646472571,"max_nodes":1000,"issued_to":"elasticsearch","issuer":"elasticsearch","start_date_in_millis":-1,"hkey":"1ddad8f727d1382a8616228647b077c7eb0258a1669db993b3c0c453c470b000"},"cluster_stats":{"timestamp":1599054541305,"status":"yellow","indices":{"count":1,"shards":{"total":5,"primaries":5,"replication":0.0,"index":{"shards":{"min":5,"max":5,"avg":5.0},"primaries":{"min":5,"max":5,"avg":5.0},"replication":{"min":0.0,"max":0.0,"avg":0.0}}},"docs":{"count":1,"deleted":0},"store":{"size_in_bytes":324,"throttle_time_in_millis":0},"fielddata":{"memory_size_in_bytes":0,"evictions":0},"query_cache":{"memory_size_in_bytes":0,"total_count":0,"hit_count":0,"miss_count":0,"cache_size":0,"cache_count":0,"evictions":0},"completion":{"size_in_bytes":0},"segments":{"count":1,"memory_in_bytes":2042,"terms_memory_in_bytes":1510,"stored_fields_memory_in_bytes":312,"term_vectors_memory_in_bytes":0,"norms_memory_in_bytes":128,"points_memory_in_bytes":0,"doc_values_memory_in_bytes":92,"index_writer_memory_in_bytes":0,"version_map_memory_in_bytes":0,"fixed_bit_set_memory_in_bytes":0,"max_unsafe_auto_id_timestamp":-1,"file_sizes":{}}},"nodes":{"count":{"total":1,"data":1,"coordinating_only":0,"master":1,"ingest":1},"versions":["5.6.16"],"os":{"available_processors":72,"allocated_processors":32,"names":[{"name":"Linux","count":1}],"mem":{"total_in_bytes":405449981952,"free_in_bytes":135116738560,"used_in_bytes":270333243392,"free_percent":33,"used_percent":67}},"process":{"cpu":{"percent":0},"open_file_descriptors":{"min":733,"max":733,"avg":733}},"jvm":{"max_uptime_in_millis":81703,"versions":[{"version":"1.8.0_201","vm_name":"OpenJDK 64-Bit Server VM","vm_version":"25.201-b09","vm_vendor":"Oracle Corporation","count":1}],"mem":{"heap_used_in_bytes":553007328,"heap_max_in_bytes":2075918336},"threads":139},"fs":{"total_in_bytes":21464350720,"free_in_bytes":20901429248,"available_in_bytes":20901429248},"plugins":[{"name":"ingest-user-agent","version":"5.6.16","description":"Ingest processor that extracts information from a user agent","classname":"org.elasticsearch.ingest.useragent.IngestUserAgentPlugin","has_native_controller":false},{"name":"ingest-geoip","version":"5.6.16","description":"Ingest processor that uses looksup geo data based on ip adresses using the Maxmind geo database","classname":"org.elasticsearch.ingest.geoip.IngestGeoIpPlugin","has_native_controller":false},{"name":"x-pack","version":"5.6.16","description":"Elasticsearch Expanded Pack Plugin","classname":"org.elasticsearch.xpack.XPackPlugin","has_native_controller":true}],"network_types":{"transport_types":{"netty4":1},"http_types":{"netty4":1}}}},"cluster_state":{"status":"yellow","version":43,"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","master_node":"_tIBWV9GS4uLw3ShKw-y-w","nodes":{"_tIBWV9GS4uLw3ShKw-y-w":{"name":"_tIBWV9","ephemeral_id":"Lrp_21m9QXyPgz_gHEStDQ","transport_address":"172.17.0.23:9300","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}}}},"stack_stats":{"xpack":{"security":{"available":true,"enabled":false},"watcher":{"available":true,"enabled":true,"execution":{"actions":{"_all":{"total":0,"total_time_in_ms":0}}},"count":{"total":0,"active":0}},"monitoring":{"available":true,"enabled":true,"enabled_exporters":{"local":1}},"graph":{"available":true,"enabled":true},"ml":{"available":true,"enabled":true,"jobs":{"_all":{"count":0,"detectors":{"total":0.0,"min":0.0,"avg":0.0,"max":0.0},"model_size":{"total":0.0,"min":0.0,"avg":0.0,"max":0.0}}},"datafeeds":{"_all":{"count":0}}}}}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.308Z","type":"node_stats","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"node_stats":{"node_id":"_tIBWV9GS4uLw3ShKw-y-w","node_master":true,"mlockall":false,"indices":{"docs":{"count":1},"store":{"size_in_bytes":324,"throttle_time_in_millis":0},"indexing":{"index_total":66,"index_time_in_millis":934,"throttle_time_in_millis":0},"search":{"query_total":15,"query_time_in_millis":65},"query_cache":{"memory_size_in_bytes":0,"hit_count":0,"miss_count":0,"evictions":0},"fielddata":{"memory_size_in_bytes":0,"evictions":0},"segments":{"count":1,"memory_in_bytes":2042,"terms_memory_in_bytes":1510,"stored_fields_memory_in_bytes":312,"term_vectors_memory_in_bytes":0,"norms_memory_in_bytes":128,"points_memory_in_bytes":0,"doc_values_memory_in_bytes":92,"index_writer_memory_in_bytes":0,"version_map_memory_in_bytes":0,"fixed_bit_set_memory_in_bytes":0},"request_cache":{"memory_size_in_bytes":0,"evictions":0,"hit_count":0,"miss_count":0}},"os":{"cpu":{"load_average":{"1m":377.89,"5m":498.09,"15m":413.31}},"cgroup":{"cpuacct":{"control_group":"/","usage_nanos":73146970553},"cpu":{"control_group":"/","cfs_period_micros":100000,"cfs_quota_micros":-1,"stat":{"number_of_elapsed_periods":0,"number_of_times_throttled":0,"time_throttled_nanos":0}}}},"process":{"open_file_descriptors":733,"max_file_descriptors":120000,"cpu":{"percent":0}},"jvm":{"mem":{"heap_used_in_bytes":553007328,"heap_used_percent":26,"heap_max_in_bytes":2075918336},"gc":{"collectors":{"young":{"collection_count":3,"collection_time_in_millis":1401},"old":{"collection_count":1,"collection_time_in_millis":173}}}},"thread_pool":{"bulk":{"threads":23,"queue":0,"rejected":0},"generic":{"threads":5,"queue":0,"rejected":0},"get":{"threads":0,"queue":0,"rejected":0},"index":{"threads":0,"queue":0,"rejected":0},"management":{"threads":4,"queue":0,"rejected":0},"search":{"threads":19,"queue":0,"rejected":0},"watcher":{"threads":0,"queue":0,"rejected":0}},"fs":{"total":{"total_in_bytes":21464350720,"free_in_bytes":20901429248,"available_in_bytes":20901429248}}}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"STARTED","primary":true,"node":"_tIBWV9GS4uLw3ShKw-y-w","relocating_node":null,"shard":2,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"UNASSIGNED","primary":false,"node":null,"relocating_node":null,"shard":2,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"STARTED","primary":true,"node":"_tIBWV9GS4uLw3ShKw-y-w","relocating_node":null,"shard":1,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"UNASSIGNED","primary":false,"node":null,"relocating_node":null,"shard":1,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"STARTED","primary":true,"node":"_tIBWV9GS4uLw3ShKw-y-w","relocating_node":null,"shard":3,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"UNASSIGNED","primary":false,"node":null,"relocating_node":null,"shard":3,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"STARTED","primary":true,"node":"_tIBWV9GS4uLw3ShKw-y-w","relocating_node":null,"shard":4,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"UNASSIGNED","primary":false,"node":null,"relocating_node":null,"shard":4,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"STARTED","primary":true,"node":"_tIBWV9GS4uLw3ShKw-y-w","relocating_node":null,"shard":0,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.310Z","type":"shards","state_uuid":"Xba5ngoiSnq7IIrbOjf3Aw","shard":{"state":"UNASSIGNED","primary":false,"node":null,"relocating_node":null,"shard":0,"index":"my-index"}}",
    "{"cluster_uuid":"S-9H2HlZTyqGcM78RloGPg","timestamp":"2020-09-02T13:49:01.311Z","type":"index_recovery","source_node":{"uuid":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9","attributes":{"ml.max_open_jobs":"10","ml.enabled":"true"}},"index_recovery":{"shards":[{"index_name":"my-index","id":2,"type":"EMPTY_STORE","stage":"DONE","primary":true,"start_time_in_millis":1599054540966,"stop_time_in_millis":1599054541005,"total_time_in_millis":38,"source":{},"target":{"id":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9"},"index":{"size":{"total_in_bytes":0,"reused_in_bytes":0,"recovered_in_bytes":0,"percent":"0.0%"},"files":{"total":0,"reused":0,"recovered":0,"percent":"0.0%"},"total_time_in_millis":0,"source_throttle_time_in_millis":0,"target_throttle_time_in_millis":0},"translog":{"recovered":0,"total":0,"percent":"100.0%","total_on_start":0,"total_time_in_millis":32},"verify_index":{"check_index_time_in_millis":0,"total_time_in_millis":0}},{"index_name":"my-index","id":1,"type":"EMPTY_STORE","stage":"DONE","primary":true,"start_time_in_millis":1599054540970,"stop_time_in_millis":1599054541010,"total_time_in_millis":39,"source":{},"target":{"id":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9"},"index":{"size":{"total_in_bytes":0,"reused_in_bytes":0,"recovered_in_bytes":0,"percent":"0.0%"},"files":{"total":0,"reused":0,"recovered":0,"percent":"0.0%"},"total_time_in_millis":0,"source_throttle_time_in_millis":0,"target_throttle_time_in_millis":0},"translog":{"recovered":0,"total":0,"percent":"100.0%","total_on_start":0,"total_time_in_millis":17},"verify_index":{"check_index_time_in_millis":0,"total_time_in_millis":0}},{"index_name":"my-index","id":3,"type":"EMPTY_STORE","stage":"DONE","primary":true,"start_time_in_millis":1599054540980,"stop_time_in_millis":1599054541018,"total_time_in_millis":37,"source":{},"target":{"id":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9"},"index":{"size":{"total_in_bytes":0,"reused_in_bytes":0,"recovered_in_bytes":0,"percent":"0.0%"},"files":{"total":0,"reused":0,"recovered":0,"percent":"0.0%"},"total_time_in_millis":0,"source_throttle_time_in_millis":0,"target_throttle_time_in_millis":0},"translog":{"recovered":0,"total":0,"percent":"100.0%","total_on_start":0,"total_time_in_millis":26},"verify_index":{"check_index_time_in_millis":0,"total_time_in_millis":0}},{"index_name":"my-index","id":4,"type":"EMPTY_STORE","stage":"DONE","primary":true,"start_time_in_millis":1599054541076,"stop_time_in_millis":1599054541131,"total_time_in_millis":55,"source":{},"target":{"id":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9"},"index":{"size":{"total_in_bytes":0,"reused_in_bytes":0,"recovered_in_bytes":0,"percent":"0.0%"},"files":{"total":0,"reused":0,"recovered":0,"percent":"0.0%"},"total_time_in_millis":0,"source_throttle_time_in_millis":0,"target_throttle_time_in_millis":0},"translog":{"recovered":0,"total":0,"percent":"100.0%","total_on_start":0,"total_time_in_millis":36},"verify_index":{"check_index_time_in_millis":0,"total_time_in_millis":0}},{"index_name":"my-index","id":0,"type":"EMPTY_STORE","stage":"DONE","primary":true,"start_time_in_millis":1599054540989,"stop_time_in_millis":1599054541049,"total_time_in_millis":59,"source":{},"target":{"id":"_tIBWV9GS4uLw3ShKw-y-w","host":"172.17.0.23","transport_address":"172.17.0.23:9300","ip":"172.17.0.23","name":"_tIBWV9"},"index":{"size":{"total_in_bytes":0,"reused_in_bytes":0,"recovered_in_bytes":0,"percent":"0.0%"},"files":{"total":0,"reused":0,"recovered":0,"percent":"0.0%"},"total_time_in_millis":0,"source_throttle_time_in_millis":0,"target_throttle_time_in_millis":0},"translog":{"recovered":0,"total":0,"percent":"100.0%","total_on_start":0,"total_time_in_millis":31},"verify_index":{"check_index_time_in_millis":0,"total_time_in_millis":0}}]}}",
    "{"name":"Frantisek"}"]>
	at com.hazelcast.jet.elastic.ElasticClientsTest.given_clientAsString_whenReadFromElasticSource_thenFinishSuccessfully(ElasticClientsTest.java:61)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:114)
	at com.hazelcast.test.FailOnTimeoutStatement$CallableStatement.call(FailOnTimeoutStatement.java:106)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

Standard output:
```
2020-09-02 13:49:03,275 [ INFO] [hz.stupefied_euclid.cached.thread-19] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.3-SNAPSHOT] Start execution of job '04e8-03d9-0480-0001', execution 04e8-03d9-0481-0001 from coordinator [127.0.0.1]:5701
2020-09-02 13:49:03,276 [DEBUG] [hz.stupefied_euclid.jet.blocking.thread-0] [c.h.j.e.i.E.elasticSource#0]: [127.0.0.1]:5701 [jet] [4.3-SNAPSHOT] init
2020-09-02 13:49:04,029 [DEBUG] [hz.stupefied_euclid.jet.blocking.thread-0] [c.h.j.e.i.E.elasticSource#0]: [127.0.0.1]:5701 [jet] [4.3-SNAPSHOT] Initialized scroll with scrollId DnF1ZXJ5VGhlbkZldGNoBgAAAAAAAAALFl90SUJXVjlHUzR1THczU2hLdy15LXcAAAAAAAAADRZfdElCV1Y5R1M0dUx3M1NoS3cteS13AAAAAAAAAAwWX3RJQldWOUdTNHVMdzNTaEt3LXktdwAAAAAAAAAOFl90SUJXVjlHUzR1THczU2hLdy15LXcAAAAAAAAAEBZfdElCV1Y5R1M0dUx3M1NoS3cteS13AAAAAAAAAA8WX3RJQldWOUdTNHVMdzNTaEt3LXktdw==, total results , 16
2020-09-02 13:49:04,240 [DEBUG] [hz.stupefied_euclid.jet.blocking.thread-0] [c.h.j.e.i.E.elasticSource#0]: [127.0.0.1]:5701 [jet] [4.3-SNAPSHOT] Succeeded clearing 6 scrolls
2020-09-02 13:49:04,267 [DEBUG] [hz.stupefied_euclid.jet.blocking.thread-0] [c.h.j.i.JobExecutionService]: [127.0.0.1]:5701 [jet] [4.3-SNAPSHOT] Execution of job '04e8-03d9-0480-0001', execution 04e8-03d9-0481-0001 completed
```

--- a/extensions/elasticsearch/elasticsearch-5/src/test/java/com/hazelcast/jet/elastic/CommonElasticSourcesTest.java
+++ b/extensions/elasticsearch/elasticsearch-5/src/test/java/com/hazelcast/jet/elastic/CommonElasticSourcesTest.java
@@ -101,6 +101,7 @@ public abstract class CommonElasticSourcesTest extends BaseElasticTest {
 
         BatchSource<String> source = ElasticSources.elastic(
                 elasticClientSupplier(),
+                () -> new SearchRequest("my-index"),
                 hit -> (String) hit.getSourceAsMap().get("name")
         );
 

--- a/extensions/elasticsearch/elasticsearch-5/src/test/java/com/hazelcast/jet/elastic/ElasticClientsTest.java
+++ b/extensions/elasticsearch/elasticsearch-5/src/test/java/com/hazelcast/jet/elastic/ElasticClientsTest.java
@@ -22,6 +22,7 @@ import com.hazelcast.jet.JetTestInstanceFactory;
 import com.hazelcast.jet.config.JetConfig;
 import com.hazelcast.jet.pipeline.Pipeline;
 import com.hazelcast.jet.pipeline.Sinks;
+import org.elasticsearch.action.search.SearchRequest;
 import org.elasticsearch.search.SearchHit;
 import org.junit.After;
 import org.junit.Test;
@@ -53,6 +54,7 @@ public class ElasticClientsTest extends BaseElasticTest {
         Pipeline p = Pipeline.create();
         p.readFrom(ElasticSources.elastic(
                 () -> ElasticClients.client(httpHostAddress),
+                () -> new SearchRequest("my-index"),
                 SearchHit::getSourceAsString)
         ).writeTo(Sinks.list(results));
 

